PythonRegression/tests/features/steps/stitching_steps.py

```diff
@@ -0,0 +1,130 @@
+from aloe import step, world
+from util.test_logic import api_test_logic as tests
+from util.transaction_utils import transaction_logic as transactions
+from util import static_vals as static
+import threading
+import random
+
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+world.config = {}
+world.responses = {}
+
+TIMEOUT_ERROR = "Thread has not completed, request has timed out"
+
+side_tangle_address = b'SIDE9TANGLE9999999999999999999999999999999999999999999999999999999999999999999999'
+stitching_addresss = b'STITCHING9TRANSACTIONS99999999999999999999999999999999999999999999999999999999999'
+referencing_address = b'REFERENCES9STITCHING9TRANSACTION9999999999999999999999999999999999999999999999999'
+
+@step(r'a stitching transaction is issued on "([^"]*)" with the tag "([^"]*)"')
+def issue_stitching_transaction(step,node,tag):
+    world.config['nodeId'] = node
+    world.config['stitchTag'] = tag
+
+    api = tests.prepare_api_call(node)
+    logger.info('Finding Transactions')
+    side_tangle_transaction = random.choice(static.SIDE_TANGLE_TRANSACTIONS)
+    gtta_transactions = api.get_transactions_to_approve(depth=3)
+
+    trunk = side_tangle_transaction
+    branch = gtta_transactions['branchTransaction']
+    
+    logger.debug('Trunk: ' + str(trunk))
+    logger.debug('Branch: ' + str(branch))
+    
+    bundle = transactions.create_transaction_bundle(stitching_addresss,tag,0)
+    trytes = bundle[0].as_tryte_string()
+    sent_transaction = api.attach_to_tangle(trunk, branch, [trytes], 14)
+    api.broadcast_and_store(sent_transaction.get('trytes'))
+
+    #Finds transaction hash and stores it in world
+    bundlehash = api.find_transactions(bundles=[bundle.hash])
+    world.responses['stitchTransaction'] = bundlehash['hashes'][0]
+
```

Comment:
This is my intention for a future PR that will encompass moving reusable code to utility functions to reduce the overall code volume in each file. All transaction creation, bundling and issuing will be done using these modules, but since they will also be used by other steps outside the scope of this PR, I chose to keep it as a separate patch.